mmv1/third_party/terraform/utils/self_link_helpers.go

```diff
@@ -169,3 +169,15 @@ func GetRegionFromRegionSelfLink(selfLink string) string {
 	}
 	return selfLink
 }
+
+// This function supports selflinks that have regions and locations in their paths
+func GetRegionFromRegionalSelfLink(selfLink string) string {
+	re := regexp.MustCompile("projects/[a-zA-Z0-9-]*/[locations/regions]+/([a-zA-Z0-9-]*)")
```

Comment:
should this be `(?:locations|regions)` instead of` [locations/regions]+`?

Reply:
Is there a difference? I don't see one.

![image](https://user-images.githubusercontent.com/25100905/122615224-12848c80-d080-11eb-8e24-6ca525dd9038.png)

Reply:
`[locations/regions]+` matches 1+ characters that are each present in the set between the brackets; `(?:locations|regions)` matches either the word "locations" or the word "regions"

Reply:
Fixed